container-images/mds-daily/package.json

```diff
@@ -8,8 +8,6 @@
     "dist/"
   ],
   "scripts": {
-    "bundle": "webpack --mode=production --env.npm_package_name=${npm_package_name} --env.npm_package_version=${npm_package_version}",
-    "image": "yarn bundle && ../../bin/build-helper buildImage mds-daily:${npm_package_version}",
```

Comment:
Is this just because we don't deploy these things to kube? Is it possible that we will ever need to bundle to deploy updates to AWS Lambda?

Reply:
It's because these packages are totally hosed as of MDS 1.0.0, and we removed the build scripts for the normal packages, but not the container images. May be re-added at some point, but these packages not functioning blocks running a top-level `yarn image`, and we need to be able to readily deploy Taxi